### ISSUE TITLE
Focus not Working

### ISSUE THREAD

**Issue by caiocs** [SceneView/sceneform-android#200]
### Discussed in https://github.com/SceneView/sceneform-android/discussions/199

<div type='discussions-op-text'>

<sup>Originally posted by **caiocs** November 11, 2021</sup>
Hello, 
I was testing your samples and the Augmented Images was without the auto focus. I put the config.setFocusMode(Config.FocusMode.AUTO) inside the onSessionConfiguration method. But the camera still without the auto focus. Am I doing something wrong? </div>

**Comment by ThomasGorisse** [comment]
ARCore open bug here: https://github.com/google-ar/arcore-android-sdk/issues/1300

**Comment by kustraslawomir** [comment]
workaround: 
```
 session.apply {
                resume()
                pause()
                resume()
            }
```

**Comment by ThomasGorisse** [comment]
Yes it's work around but it should not be used due to the time it takes to resume an ARCore session.
Could you try the latest ARCore version to see if it has been fixed?

**Comment by devbridie** [comment]
My understanding is that this should be fixed in 1.30; please let me know if that isn't the case.

**Comment by ThomasGorisse** [comment]
@devbridie That's would be a great news.
Thanks for letting us know.
